views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -120,6 +120,17 @@
         [ ] {{__ "square brackets"}}
     </label>
 </div>
+<div>
+    <label class="panel">
+        <input name="tool_integral" type="checkbox" {{#if tool_integral}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span>
+        {{__ "Indefinite integral"}}
```

Comment:
Can the integral have its symbol displayed here?
`&#x222b; {{__ "Indefinite integral"}}`